teacher.rb

```diff
@@ -0,0 +1,11 @@
+require './person'
+
+class Teacher
+  def initialize(specialization)
+    @specialization = specialization
+  end
+
```

Comment:
- [ ] As stated in the requirements, this class should inherit from the `Person` class. Kindly take note of my comments in the `students` class file when inheriting with parameers 👍 

### Instructions:
![image](https://github.com/Mar12358/school-library-people/assets/50990925/77356e42-2694-47df-afa9-6819fbf00250)